Procfile

```diff
@@ -1 +1 @@
-worker: python app.py
\ No newline at end of file
+worker: python cli.py update
```

Comment:
I believe this will break our deploy to heroku.

The CLI should something that is invoked manually from the command line.

Reply:
I can change this. But I *believe* it does not break the deployment. I can make a test on my heroku account before merge.